pkg/controller/constrainttemplate/constrainttemplate_controller.go

```diff
@@ -331,6 +331,7 @@ func (r *ReconcileConstraintTemplate) Reconcile(ctx context.Context, request rec
 	status.Status.TemplateUID = ct.GetUID()
 	status.Status.ObservedGeneration = ct.GetGeneration()
 	status.Status.Errors = nil
+	status.Status.VAPGenerationStatus = statusv1beta1.VAPGenerationStatus{}
```

Comment:
This appears to have the same problem of wiping the generation status that constraints had.

Reply:
Fixed it. PTAL @maxsmythe